docs/EN_US/ContainerizedHPCC/ContainerizedMods/ConfigureValues.xml

```diff
@@ -1195,358 +1195,791 @@ thor:
+          by running an ecl job that requires using mutual TLS (using local client certificate):</para>
+
+          ```
+          ecl run --ssl hthor helm/examples/certmanager/localhttpcall.ecl
+          ```
+
+          <para>Note that for the HTTPCALL in our ecl example the url now starts with "mtls:"
+          this tells HTTPCALL/SOAPCALL to use mutual TLS, using the local client certificate,
+          and to verify the server using the local certificate authority certificate.</para>
+
+          <para>You should see a result similar to this, possibly with some additional headers:</para>
+
+          ```xml
+          <Result>
+            <Dataset name='localHttpEchoResult'>
+              <Row>
+                <Method>GET</Method>
+                <UrlPath>/WsSmc/HttpEcho</UrlPath>
+                <UrlParameters>name=doe,joe&amp;number=1</UrlParameters>
+                <Headers>
+                  <Header>Accept-Encoding: gzip, deflate</Header>
+                  <Header>Accept: text/xml</Header>
+                </Headers>
+              </Row>
+            </Dataset>
+          </Result>
+          ```
+        </sect4>
 
-    <sect2 id="HPCCContainer_Certificates" role="brk">
-      <title>Certificates</title>
+        <sect4 id="RemoteMutualTLS">
+          <title>Remote Mutual TLS</title>
 
-      <para>HPCC Systems containerized deployments support comprehensive
-      certificate management through Helm chart configuration. This includes
-      support for multiple certificate issuers, domain management, and
-      advanced certificate features for secure communication between
-      components.</para>
+          <para>This walkthrough demonstrates using a single Hashicorp Vault
+          PKI Certificate authority to establish a zone of trust between two separate
+          HPCC environments. cert-manager continues to handle the backing Kubernetes
+          resources. For this example each HPCC environment is in a separate kubernetes
+          namespace: hpcc1 and hpcc2, but in production they could be in different
+          Kubernetes clusters, or even in different cloud subscriptions.</para>
 
-      <para>The certificate system integrates with cert-manager to provide
-      automated certificate generation and renewal for both internal (local,
-      i.e., communication between HPCC components within the same cluster or
-      namespace) and external (public/remote) communications.</para>
+          <para>
+          Here we have hpcc2.roxie making a call to hpcc1.roxie in another 
+          environment, and we want them to trust each other. There are two main configuration
+          customizations needed to set up this zone
+          of trust. Each of the two HPCC environments have these changes.</para> 
 
-      <sect3 id="Container_Certificates">
-        <title>Enable Certificates</title>
+          <para>First, the remote issuer is enabled and its spec set to use the Vault.
```

Comment:
Question: is "spec" the actual term or is it a truncation of specification - or is it even the correct term and should it be something like 'value' in terms of the yaml-configuration file value?